src/postgres_mcp/server.py

```diff
@@ -10,6 +10,7 @@
 from .dta.dta_tools import DTATool
 from .dta.safe_sql import SafeSqlDriver
 from .dta.sql_driver import SqlDriver
+from .orchestration.database_health import DatabaseHealthTool
```

Comment:
How about we just name the package `database_health` without `orchestration` as a prefix?

Reply:
done